hints/duplicate-integer.md

```diff
@@ -1,15 +1,16 @@
 <br>
 <details class="hint-accordion">  
     <summary>Recommended Time & Space Complexity</summary>
-    <p>You should aim for a solution as good or better than <code>O(n)</code> time and <code>O(n)</code> space, where <code>n</code> is the size of the input array.
+    <p>
+    You should aim for a solution with <code>O(n)</code> time and <code>O(n)</code> space, where <code>n</code> is the size of the input array.
     </p>
 </details>
 
 <br>
 <details class="hint-accordion">  
     <summary>Hint 1</summary>
     <p>
-    A brute force solution would be to check every element against every other element in the array. This would be an <code>O(n^2)</code> solution. Can you think of a better way?
+    A brute force solution would be to check every element against every other element in the array. This would be an <code>O(n<sup>2</sup>)</code> solution. Can you think of a better way?
```

Comment:
Unfortunately, there's a limitation in the way my site renders markdown. The <sup></sup> wont work as intended so I would leave this as-is for now. And do the same for all other files.

This isn't your fault. I appreciate you trying to improve this though!

Reply:
Ok, I will keep that as it is in all the files.